cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/mazha/CouchClient.java

```diff
@@ -52,8 +60,28 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocket;
+import javax.net.ssl.SSLSocketFactory;
+
 public class CouchClient {
 
+    private static final List<HttpConnectionRequestInterceptor> DEFAULT_REQUEST_INTERCEPTORS =
+            new ArrayList<HttpConnectionRequestInterceptor>();
+
+    private static final UserAgentInterceptor USER_AGENT_INTERCEPTOR =
```

Comment:
Can `USER_AGENT_INTERCEPTOR` and `TLS_INTERCEPTOR` go into the `static` block so they don't pollute the top-level fields?

Reply:
Yes, done in 72fef696319034370794c5137617a11e5c9bea81